src/Project/Controller/ProjectControllerTrait.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace CultuurNet\ProjectAanvraag\Project\Controller;
+
+use CultuurNet\ProjectAanvraag\Core\Exception\MissingRequiredFieldsException;
+
+trait ProjectControllerTrait
```

Comment:
I would calls this `ValidateFields` or something similar.

Reply:
I orginallly called it `ProjectControllerRequest` but the `ci` demanded that it should have a name ending with `Trait`, so I just used the same name for Clarity. 
ValidateFieldsTrait could offcourse be a good alternative.